src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.MovieSessionResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.mapper.MovieSessionDtoMapper;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    @PostMapping
+    public MovieSessionResponseDto add(
+            @RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        return movieSessionDtoMapper.toDto(movieSessionService
+                .add(movieSessionDtoMapper.toModel(movieSessionRequestDto)));
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> getAvailableMovieSession(
+            @RequestParam("moveId") Long movieId,
+            @RequestParam("date") @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date) {
```

Comment:
```suggestion
            @RequestParam Long movieId,
            @RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date) {
```
no difference between names passed variables and custom name request parameters